rmf_building_map_tools/building_map_model_downloader/building_map_model_downloader.py

```diff
@@ -109,7 +109,11 @@ def download_models(
     # Ignition fuel tools can only download to this folder, so we set the
     # model path to it
     if fuel_tools:
-        model_path = "~/.ignition/fuel/"
+        if export_path is not None:
+            model_path = export_path
+            ign = False
```

Comment:
Wouldn't this revert back to using `http` to download the models after using both the `-f` and `-e` flags?

Reply:
Hello Aaron. No it would not. The `ign=False` is just so that the it can search the folder specified by `-e` for missing models. The models will still be downloaded using the new Ignition Fuel tools. I've tested it out in this [PR](https://github.com/open-rmf/rmf_demos/pull/57) for `rmf_demos` to check that its the case.

Reply:
Thanks for explaining, didn't catch these lines, https://github.com/nicholas-gs/rmf_traffic_editor/blob/fix/building_map_tools/rmf_building_map_tools/building_map_model_downloader/building_map_model_downloader.py#L154-L157